server/plugins/oidc/src/main/java/com/walmartlabs/concord/server/plugins/oidc/OidcAuthenticationHandler.java

```diff
@@ -70,7 +70,7 @@ public AuthenticationToken createToken(ServletRequest request, ServletResponse r
         String header = req.getHeader(AUTHORIZATION_HEADER);
         if (header != null) {
             String[] as = header.split(" ");
-            if (as.length != 2 || !as[0].equals(HEADER_PREFIX)) {
+            if (as.length != 2 || !HEADER_PREFIX.equals(as[0])) {
```

Comment:
Cannot be null.

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behaviors is otherwise equivalent, there is no cost to this change.